articles/openshift/tutorial-create-cluster.md

```diff
@@ -22,7 +22,7 @@ If you choose to install and use the CLI locally, this tutorial requires that yo
 
 ### Verify your permissions
 
-To create an Azure Red Hat OpenShift cluster, verify the following permissions on your Azure account and user:
+To create an Azure Red Hat OpenShift cluster, verify the following permissions on your Azure subscription, AAD user, or service principal:
```

Comment:
I was on the call with the customer and they were confused about the meaning of `account`.  I think this better describes what we are actually attempting to do.  The subscription needs it or the user or in their case they are using a service principal.  I'll defer to @ms-jasondel's Azure expertise for the wording.

Reply:
I'm looking for the right MSFT documentation to reference here, but this article describes the issue this customer was having: https://lnx.azurewebsites.net/directory-roles-for-azure-ad-service-principal/

The verbiage above should be Azure Identity (not account and not subscription) as you only assign rights to an Identity.  *If* that identity is an interactive AAD User, all AAD Users implicitly have Directory Read rights, which is what I believe https://github.com/Azure/ARO-RP/blob/master/python/az/aro/azext_aro/_aad.py#L55 requires.  However, SPs do not have this permission by default and I believe an Administrator needs to grant Admin Consent to this SP in order for it to have this permission.

Reply:
I have succesfully tested this scenario, logging in as a Service Principal (az login --service-principal -u xxx -p xx -t xxxx) and executing az aro create.  The SP xxx has Azure Active Directory Graph: Application.ReadWrite.OwnedBy and Microsoft Graph: User.Read permissions granted admin consent.  I'm currently validating whether it requires BOTH of those permissions or just the later.  Upon confirmation, I'll append a *NOTE* to the doc above highlighting this scenario.

Reply:
We should also make clear in the docs that a Service Principal passed to --client-id must be unique per cluster, they cannot share SPs or you will get the following error:

"code": "DuplicateClientID",
"message": "The provided client ID 'xxxx' is already in use by a cluster.